### ISSUE TITLE
Fails to start after being install ./install-ubuntu on ubuntu 15.10

### ISSUE THREAD

**Issue by Korons** [ilius/starcal#24]
I cloned the git repo and installed the program with ```sudo ./install-ubuntu``` and then ran ```starcal3``` the program exited with 

```
Traceback (most recent call last):
  File "/usr/share/starcal3/scal3/ui_gtk/starcal.py", line 58, in <module>
    from scal3 import ui
  File "/usr/share/starcal3/scal3/ui.py", line 651, in <module>
    eventTrash = event_lib.EventTrash.load()
  File "/usr/share/starcal3/scal3/s_object.py", line 208, in load
    raise FileNotFoundError('%s : file not found'%_file)
FileNotFoundError: /home/USER/.starcal3/event/trash.json : file not found
```

If I create the file /home/USER/.starcal3/event/trash.json and try to run the program it fails with 

```
Traceback (most recent call last):
  File "/usr/share/starcal3/scal3/ui_gtk/starcal.py", line 58, in <module>
    from scal3 import ui
  File "/usr/share/starcal3/scal3/ui.py", line 651, in <module>
    eventTrash = event_lib.EventTrash.load()
  File "/usr/share/starcal3/scal3/s_object.py", line 214, in load
    raise e
  File "/usr/share/starcal3/scal3/s_object.py", line 211, in load
    data = jsonToData(jsonStr)
  File "/usr/lib/python3.4/json/__init__.py", line 318, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python3.4/json/decoder.py", line 343, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python3.4/json/decoder.py", line 361, in raw_decode
    raise ValueError(errmsg("Expecting value", s, err.value)) from None
ValueError: Expecting value: line 1 column 1 (char 0)
```

**Comment by ilius** [comment]
If you have not added any events with StarCaendar 3.x, quit the program, run `rm -R ~/.starcal3` and run the program again.
I guess that could have happened because of incomplete importing from StarCalendar 2.x

**Comment by Korons** [comment]
I ran ```rm -R ~/.starcal3``` and re-downloaded and reinstalled starcal3 (from the git repo) and am getting the same error. However I installed the deb from [here](https://github.com/ilius/starcal/releases/download/3.0.1/starcal3_3.0.1-1_all.deb) and it now works fine. 

The deb that was built with ```sudo ./install-ubuntu``` has a sha256 sum of 87ce35795eecccbb9bf7cdc123e7826914639c81487255c4d6e01e2fa108666d 

and the version I downloaded has a sha256 sum of 6e6bd20d64484433c7df5005d2f42f75d80c1693f69bcea1313945fa713bdcb8

**Comment by ilius** [comment]
This bug is fixed in tag `3.0.2`, commit 5d057fbf78e89233c81a1a55cd764177a0e116b2
